install/kubernetes/helm/istio/charts/certmanager/templates/deployment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "certmanager.name" . }}
```

Comment:
In other templates I think we use defined name, we want to reduce arbitrary customizations, it's harder to debug.